pystiche_papers/johnson_alahi_li_2016/_loss.py

```diff
@@ -54,6 +54,31 @@ def content_loss(
     layer: str = "relu2_2",
     score_weight: Optional[float] = None,
 ) -> ops.FeatureReconstructionOperator:
+    r"""Content_loss from :cite:`JAL2016`.
+
+    Args:
+        impl_params: If ``True``, use the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper.
+        instance_norm: If ``True``, use :class:`~torch.nn.InstanceNorm2d` rather than
+            :class:`~torch.nn.BatchNorm2d` as described in the paper.
+        style: Optional style for selecting the optimization parameters for the
+            replication of the given pre-trained models from the original repository.
+            See score_weight for details.
+        multi_layer_encoder: Pretrained :class:`~pystiche.enc.MultiLayerEncoder`. If
+            omitted, the default
+            :func:`~pystiche_papers.johnson_alahi_li_2016.multi_layer_encoder` is used.
+        layer: Layer from which the encodings of the ``multi_layer_encoder`` should be
+            taken. Defaults to "relu2_2".
+        score_weight: Score weight of the operator. If omitted, the score_weight is
+            determined with respect to ``style`` and ``instance_norm``. For details see
+            :ref:`here <table-hyperparameters>`.
+
+    The parameter ``impl_params`` is passed to the
+    :func:`~pystiche_papers.johnson_alahi_li_2016.multi_layer_encoder` and a getter
+    function to get the content ``score_weight`` if omitted.
```

Comment:
I'm probably changing my mind here, but I'm not happy with that as is for two reasons:

1. I (now) think we can omit stating that `impl_params` is passed down. I think that is implicitly clear.
2. The second part does not add any information that is not already described in `score_weight` above.

Thus, I think we can leave it out. Thoughts?

Reply:
Yeah, I think you're right about that, too. Especially because of the list of all functions that are affected by the parameter, it should be clear that `impl_params` is passed down if nothing else is described.